src/components/NewGalaxyModal.js

```diff
@@ -308,11 +308,11 @@ export const NewGalaxyModalBase = Utils.withDisplayName('NewGalaxyModal')(
       const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
       return Utils.cond(
         [currentDataDisk, () => {
-          return div({ marginTop: '0.75rem' }, [
```

Comment:
the two changes involving margins in this file are correcting bugs that showed up in the console. Note that these margins are not actually in a style prop, and are floating top-level props (aka not doing anything)